pkg/commands/deploy.go

```diff
@@ -64,6 +65,12 @@ func NewRemoveCommand(commonOpts *CommonOptions) *cobra.Command {
 			if opts.clusterPlatform == platform.Unknown {
 				return fmt.Errorf("cannot autodetect the platform, and no platform given")
 			}
+			versionDetect, source := detect.FindVersion(platDetect.Discovered, commonOpts.UserPlatformVersion)
+			commonOpts.DebugLog.Printf("Version detection source: %s", source)
+			opts.clusterVersion = versionDetect.Discovered
+			if opts.clusterVersion == platform.MissingVersion {
+				return fmt.Errorf("cannot autodetect the platform version, and no version given")
+			}
```

Comment:
please pass `Version: opts.ClusterVersion` down when we call `updaters.Remove` (you may need to change `updaters.Options`)

Reply:
Ah, sorry about that. Will fix now.

Reply:
I had already updated the `Updaters.Options` but missed assigning version to it.

Reply:
> I had already updated the `Updaters.Options` but missed assigning version to it.

right, the change is fine as you did. Just pass the value and we should be done.